### ISSUE TITLE
Presistent Data

### ISSUE THREAD

**Issue by rezytijo** [0xgkd/awvs#8]
Can I configure this container using docker  volume so that the data is not lost if my VPS crashes at any time?

**Comment by 0xgkd** [comment]
> Can I configure this container using docker volume so that the data is not lost if my VPS crashes at any time?

Yes, you can use docker volume syncing some necessary data, or restart container and generate scan report to save.

**Comment by rezytijo** [comment]
How About binding volume?

which directory i should bind then?

**Comment by 0xgkd** [comment]
> How About bind mount volume?
> 
> which directory i should bind then?

When scan in progress, /home/acunetix/.acunetix/data/temp will exist some scan temp data, and once scan completed, /home/acunetix/.acunetix/data/reports will generate html and pdf format of report, /home/acunetix/.acunetix/data/scans will generate a zip compressed file contains browser.log, logfile.csv, settings.xml.

**Comment by rezytijo** [comment]
Thanks for the info, i will try that config, thanks

**Comment by 0xPrial** [comment]
I am also looking for a way to save data on restart. Anyone got the solution for this?

**Comment by rezytijo** [comment]
u should bind this directory :

> /home/acunetix/.acunetix/data/:rw

Docker Compose :
```docker-compose.yml
services:
  awvs:
    image: 0xgkd/awvs
    container_name: awvs
    restart: always
    ports:
      - "3443:3443"
    volumes:
      - acunetix-data:/home/acunetix/.acunetix/data/:rw
    # TTY dan Stdin open diperlukan agar container tidak langsung exit
    stdin_open: true 
    tty: true

volumes:
  acunetix-data:
    name: acunetix-data
```
